docs/Deploying-Pega-on-EKS.md

```diff
@@ -494,6 +488,8 @@ Configure the parameters so the pega.yaml Helm chart matches your deployment res
 
 - Access your RDS database.
 
+- Access to the ElasticSearch service deployed using SRS nodes.
```

Comment:
slight tweak to make it parallel and show SRS is recommended but optional>>
Access your ElasticSearch service (For 8.6 and later, Pega recommends deploying your service using an SRS cluster).

Reply:
done.